clients/js/test/auth.token.test.ts

```diff
@@ -56,4 +56,16 @@ if (!process.env.XTOKEN_TEST) {
         expect(collections.length).toBe(1)
     })
 
+    // test CloudClient here
```

Comment:
Probably unneeded comment?

Reply:
thank you for your dedication to quality!